clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -439,6 +439,109 @@ pos_c"
     )
 }
 
+#[test]
+fn suggest_argument_multi_values() {
+    let mut cmd = Command::new("dynamic")
+        .arg(
+            clap::Arg::new("certain_num")
+                .long("certain_num")
+                .short('Y')
+                .value_parser(["val1", "val2", "val3"])
+                .num_args(3),
+        )
+        .arg(
+            clap::Arg::new("uncertain_num")
+                .long("uncertain_num")
+                .short('N')
+                .value_parser(["val1", "val2", "val3"])
+                .num_args(1..=3),
```

Comment:
Even in tests, lets use standard naming conventions.  CLI flags should be kebab-case rather than snake_case